src/libstd/fs.rs

```diff
@@ -551,7 +551,7 @@ impl OpenOptions {
     ///
     /// This option, when true, means that writes will append to a file instead
     /// of overwriting previous contents.
-    /// Note that setting `.write(true).append(true)` has the same effect as
+    /// Note that setting `.create(true)..write(true).append(true)` has the same effect as
     /// setting only `.append(true)`.
```

Comment:
this seems inaccurate. my understanding is that `.create(true).write(true).append(true)` and `.append(true)` are not equivalent. this was explained in https://github.com/rust-lang/rust/issues/44432

Reply:
Oups, went too fast...